src/server/auth/server/testing/auth_test.go

```diff
@@ -27,54 +27,11 @@ import (
 	globlib "github.com/pachyderm/ohmyglob"
 )
 
-// aclEntry mirrors auth.ACLEntry, but without XXX_fields, which make
```

Comment:
Repos now use the same structure as the cluster role bindings, so these helpers aren't necessary.